src/simularium/types.ts

```diff
@@ -49,6 +49,8 @@ export interface EncodedTypeMapping {
 }
 
 interface TrajectoryFileInfoGeneric {
```

Comment:
What does "generic" mean here?

Reply:
Oh it's probably not an appropriate use of that word... What I meant is more like `TrajectoryFileInfoBase` or `TrajectoryFileInfoRoot` or `TrajectoryFileInfoCommon`? It just contains all the stuff that v1 and v2 versions have in common. Do you have a preference or suggestions?